.vib/redis/goss/redis.yaml

```diff
@@ -0,0 +1,27 @@
+file:
+  /opt/bitnami/redis/etc/redis.conf:
+    exists: true
+    mode: "0664"
+    contains:
+      - /port.*6379/
+      - /dir.*/bitnami/redis/data/
+      - /pidfile.*/opt/bitnami/redis/tmp/redis.pid/
+      - daemonize yes
+command:
+  check-redis-server:
+    exec: timeout 5s redis-server || true
+    exit-status: 0
+    stdout:
+      - Ready to accept connections
+  check-redis-server-ssl:
```

Comment:
I'm not sure about these 2 ssl checks, there is no related compilation or postunpack logic that requires it right?

Reply:
I've added those based on review comments:

https://github.com/bitnami/containers/pull/28158#discussion_r1149286198